### PR TITLE
Create times_of_repeated_word_in_string.c

### PR DESCRIPTION
**Pull request by Vaishnavi-iit**

### CODE REVIEW

File: 1.Practicals_C/Array_String_Structue/Strings/times_of_repeated_word_in_string.c [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#include<stdio.h>
+void main()
```
Comment (Anjali-1801) [inline 1.Practicals_C/Array_String_Structue/Strings/times_of_repeated_word_in_string.c:2]: Add #include<string.h> header file

Comment (Anjali-1801) [inline 1.Practicals_C/Array_String_Structue/Strings/times_of_repeated_word_in_string.c:2]: Declare the custom function "repeat".

**Review by Anjali-1801 — CHANGES_REQUESTED** [review]
Kindly make the required changes.

File: 1.Practicals_C/Array_String_Structue/Strings/times_of_repeated_word_in_string.c (hunk 1 above)
Comment (Vaishnavi-iit) [reply 1.Practicals_C/Array_String_Structue/Strings/times_of_repeated_word_in_string.c:4]: I have done the required changes.Thank you for suggesting...

File: use_of_fgetc.c [hunk 2]
```diff
@@ -0,0 +1,21 @@
+#include<stdio.h>
+void main()
```
Comment (Anjali-1801) [inline use_of_fgetc.c:2]: "void main()" returns nothing. So, either change it to "int main()" or remove "return 0"

File: use_of_fgetc.c [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+{
+   FILE *fp;
+   char c;
+   fp=fopen("C://DE_Data/my_file.txt","r");                 // opening an existing file in read mode
+   if(fp==NULL)
+   {
+       printf("\nCould not open file test.c");            //if file does not exist in current directory it will not create new file in read mode.
+   }
+   printf("\n");
```
Comment (Anjali-1801) [inline use_of_fgetc.c:11]: printf("\n") is not required here.

File: use_of_fgetc.c [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+   while(1)
+   {
+       c=fgetc(fp);                                        // reading the file character by character
+       if(c==EOF)
+       break;
+       printf("%c",c);                                    //printing the content of file
+   }
```
Comment (Anjali-1801) [inline use_of_fgetc.c:18]: Instead add printf("\n") here to improve the readability.

**Review by Anjali-1801 — CHANGES_REQUESTED** [review]

File: use_of_fgetc.c (hunk 4 above)
Comment (Vaishnavi-iit) [reply use_of_fgetc.c:17]: The output is now readable. Suggest further improvement.

File: 1.Practicals_C/Array_String_Structue/Strings/times_of_repeated_word_in_string.c (hunk 1 above)
Comment (Vaishnavi-iit) [reply 1.Practicals_C/Array_String_Structue/Strings/times_of_repeated_word_in_string.c:4]: Now I have declare it.Thank you.

File: use_of_fgetc.c (hunk 3 above)
Comment (Vaishnavi-iit) [reply use_of_fgetc.c]: I removed that.Thank you.

**Review by Anjali-1801 — APPROVED** [review]

**Pull request merged**